datasets/kwaliteitsmonitor/kwaliteitsmonitorWonen/v1.0.0.json

```diff
@@ -0,0 +1,155 @@
+{
+  "id": "kwaliteitsmonitorWonen",
```

Comment:
Kan deze tabel hernoemd worden naar `wonen`? Dan krijgen we `kwaliteitsmonitor/wonen` ipv. `kwaliteitsmonitor/kwaliteitsmonitorWonen`.